Add blog post: LibrePCB 0.1.4 Released (#43)
LibrePCB/LibrePCB.github.io: content/blog/2020-05-03_release_0.1.4.md

[![Search Symbol](/img/schematic_editor_search.gif)](/img/schematic_editor_search.gif)

Furthermore, all input fields for length values (line width, drill diameter,
coordinate etc.) are now displayed in the unit configured in the workspace
settings ([#684](https://github.com/LibrePCB/LibrePCB/pull/684)). The unit of
each input field can even be changed individually and gets saved and restored
automatically. And you can now enter mathematical expressions and optionally
append a unit (e.g. "mm") which will then be evaluated:

[![Search Symbol](/img/length_input_unit.gif)](/img/length_input_unit.gif)

#### Pick&Place and SVG Export

This release adds a pick&place CSV file export
([#639](https://github.com/LibrePCB/LibrePCB/pull/639)) so LibrePCB is now
also suitable to design PCBs which will be assembled by pick&place machines:

[![Pick&Place Export](/img/pickplace_export_dialog.png)](/img/pickplace_export_dialog.png)

In addition, an SVG export has been implemented for both, schematics and
boards ([#656](https://github.com/LibrePCB/LibrePCB/pull/656)). This allows to
postprocess them in a vector graphics tool for various purposes.

> Apropos file export: The BOM CSV export is now comma-separated instead of

Reviewer: ```suggestion
À propos file export: The BOM CSV export is now comma-separated instead of
```

semicolon-separated ([#678](https://github.com/LibrePCB/LibrePCB/pull/678)).
This improves compatibility with CSV readers.

### Compatibility Note

LibrePCB 0.1.4 uses exactly the same file format for projects as all previous
0.1.x releases and are thus fully compatible.

However, previous versions had a bug which allowed to have linebreaks in net
names ([#603](https://github.com/LibrePCB/LibrePCB/issues/603)). This has been

> fixed in 0.1.4, and since we expect (or hope...) no projects to be affected by

Reviewer: ```suggestion
fixed in 0.1.4, and since we expect (or hope…) no projects to be affected by
```

- Respect configured workspace length unit in editors
  ([#684](https://github.com/LibrePCB/LibrePCB/pull/684))
- Respect configured grid unit in statusbar of editors
  ([#689](https://github.com/LibrePCB/LibrePCB/pull/689))
- Improve keyboard navigation in dialogs
  ([#690](https://github.com/LibrePCB/LibrePCB/pull/690))
- Use wizard to create and switch workspace from ControlPanel
  ([#688](https://github.com/LibrePCB/LibrePCB/pull/688))
- Split workspace chooser and workspace initialization wizard
  ([#652](https://github.com/LibrePCB/LibrePCB/pull/652))
- Fix falling back to system language for untranslated strings
  ([#645](https://github.com/LibrePCB/LibrePCB/pull/645))
- Various internal code improvements and refactoring
- Many translation improvements and fixes

#### Installer:
- Change default installation directory on macOS to `/Applications`
  ([#644](https://github.com/LibrePCB/LibrePCB/pull/644))

#### New languages:
- [Czech](https://www.transifex.com/librepcb/librepcb-application/language/cz/)
  (93% translated)
- [Chinese (Taiwan)](https://www.transifex.com/librepcb/librepcb-application/language/zh_TW/)
  (99% translated)

### Download

> The release can be downloaded for all major operating systems on our download

Reviewer: ```suggestion
The release can be downloaded for all major operating systems from our download
```
